lib/attribute-parser.js

```diff
@@ -152,8 +157,10 @@ module.exports = class AttributeParser {
 
         }
         else {
+
           const exp = 'v = ' + value
-          eval(exp) // eval is evil except in this case!
+          v = vm.runInThisContext(exp)
```

Comment:
I don't use the same API here I use this one https://nodejs.org/api/vm.html#vm_vm_runincontext_code_contextifiedsandbox_options